### DISCUSSION TITLE
EA App Integration?

### DISCUSSION THREAD

**Discussion by Paradonas** [GOG-Nebula/.github#19] [Ideas]
## I can't even log in with the Origin integration available in the Galaxy 2 app
I've seen some EA integrations from [Geezmo](https://github.com/geeeezmo/galaxy-integration-eaapp) and FriendsOfGalaxy but they don't seem widely used or done yet. Neither do they have any setup instructions or a proper readme. It would be cool now that the EA app has been used over Origin for quite a while to feature and complete with the assistance of a widely used collection like this. 
/ A GOG user who recently discovered the delights of the featured steam integration fix
![GalaxyClient_8T0v2RsUkm](https://github.com/GOG-Nebula/.github/assets/90418138/ba9514fd-299d-4696-bfdb-471edbf1300b)
![GalaxyClient_GeZZD34HjT](https://github.com/GOG-Nebula/.github/assets/90418138/1771aab1-c3f5-424e-89f1-614e3bb854e3)

**Comment by Paradonas** [comment]
I also can't figure out how to use the UPlay integration. Do I just copy over the contents of src? Why is there no README? I don't have any much to any experience with python specifically soo...

**Reply by Paradonas** [reply]
I got it working after doing exactly that. Thanks guys.. 👌

**Comment by Nutzzz** [comment]
Re EA Desktop, see https://github.com/BellezaEmporium/galaxy-integration-ead .  It's an early start.  Login is successful, and your owned games are shown, but installed games may or may not show as such.  EA Desktop is opened when Play or Install is clicked.  Some discussion here: https://github.com/FriendsOfGalaxy/galaxy-integration-origin/issues/44

**Comment by ABaumher** [comment]
If there's progress on the repo mentioned above and the other members of Nebula are okay with it, i'll open a fork for Origin/EA Play and we'll see about using that repo as an Upstream. For now i've offered assistance, though i'm not sure if i'll be a ton of help without any context
